### ISSUE TITLE
Unable to install the nvdashboard extension using a supported jupyterlab version 3.3.1 

### ISSUE THREAD

**Issue by ovalerio** [rapidsai/jupyterlab-nvdashboard#123]
I am attempting to install the nvdashboard jupyterlab extension without success. I downloaded the extension using pip and installed jupyterlab using conda from the conda-forge channel.  This is the error message that I got while attempting the install.

```$ jupyter labextension install jupyterlab-nvdashboard
An error occurred.
ValueError: The extension "jupyterlab-nvdashboard" does not yet support the current version of JupyterLab.

Conflicting Dependencies:
JupyterLab              Extension        Package
>=3.3.1 <3.4.0          >=2.0.0 <3.0.0   @jupyterlab/application
>=3.3.1 <3.4.0          >=2.0.0 <3.0.0   @jupyterlab/apputils
>=5.3.1 <5.4.0          >=4.0.0 <5.0.0   @jupyterlab/coreutils
>=17.0.1 <18.0.0        >=16.4.2 <17.0.0 react
>=17.0.1 <18.0.0        >=16.9.0 <17.0.0 react-dom
See the log file for details:  /tmp/jupyterlab-debug-wdnyns59.log
```

This is the version of jupyterlab I am using:

```$ conda list | grep jupyterlab
jupyterlab                3.3.1              pyhd8ed1ab_0    conda-forge
jupyterlab-nvdashboard    0.6.0                    pypi_0    pypi
jupyterlab_pygments       0.1.2                      py_0    anaconda
jupyterlab_server         2.10.3             pyhd8ed1ab_0    conda-forge
```

**Comment by ovalerio** [comment]
Thanks Jacob,  

I will try updating the `jupyterlab-nvdashboard`  I found interesting that you didn't even have to install the extension in your test environment:

```
jupyterlab labextension install jupyterlab-nvdashboard
```
Is that step no longer needed in newer versions of the dashboard?
When I list the extensions installed it shows that the extension is enabled:

```
$ jupyter labextension list
JupyterLab v3.3.1
/gpfs/soma_fs/home/valerio/anaconda3/envs/tracking/share/jupyter/labextensions
        jupyterlab-nvdashboard v0.6.0 enabled OK (python, jupyterlab_nvdashboard)
        dask-labextension v5.2.0 enabled OK (python, dask_labextension)
        @jupyterlab/server-proxy v3.2.1 enabled OK
```

**Comment by ovalerio** [comment]
I am using the latest version of the `jupyterlab-nvdashboard` available in PyPi

```
$ pip index versions jupyterlab-nvdashboard
WARNING: pip index is currently an experimental command. It may be removed/changed in a future release without prior warning.
jupyterlab-nvdashboard (0.6.0)
Available versions: 0.6.0, 0.5.0, 0.4.0, 0.3.1, 0.3.0, 0.2.1, 0.2.0, 0.1.11200212, 0.1.11, 0.1.10, 0.1.9, 0.1.8, 0.1.7, 0.1.6, 0.1.5, 0.1.3, 0.1.2, 0.1.1, 0.1.0
  INSTALLED: 0.6.0
  LATEST:    0.6.0
```

**Comment by jacobtomlinson** [comment]
> Is that step no longer needed in newer versions of the dashboard?

That's correct, that is a Jupyter Lab 2.0 thing. You don't need it in Jupyter Lab 3.0. We don't mention it in the installation instructions.

I wonder if this is the problem. You are installing the JLv3 version via `pip` and then overwriting the JavaScript side of things with the legacy `jupyterlab labextension install` command. I expect this would install an older and incompatible version from NPM.
